lib/feature_management.rb

```diff
@@ -6,7 +6,16 @@ def self.telephony_disabled?
   def self.prefill_otp_codes?
     # In development, when SMS is disabled we pre-fill the correct codes so that
     # developers can log in without needing to configure SMS delivery.
-    Rails.env.development? && FeatureManagement.telephony_disabled?
+    # We also allow this in production on a single server that is used for load testing.
+    development_and_telephony_disabled? || prefill_otp_codes_allowed_in_production?
+  end
+
+  def self.development_and_telephony_disabled?
+    Rails.env.development? && telephony_disabled?
+  end
+
+  def self.prefill_otp_codes_allowed_in_production?
+    Figaro.env.domain_name == 'idp.pt.login.gov' && telephony_disabled?
```

Comment:
what do you think about assigning `'idp.pt.login.gov'` to a constant so we can change it easily once prod lives at `login.gov` ?